foxpuppet/foxpuppet.py

```diff
@@ -8,7 +8,11 @@
 
 
 class FoxPuppet(object):
-    """ Class that sets up the api for interacting with the Firefox browser.
+    """
+        Class that sets up the interface for interacting with the Firefox
+        browser.
+
+        :param selenium: A selenium object
```

Comment:
Let's call this a WebDriver object and link to Selenium's docs by including the type. https://seleniumhq.github.io/selenium/docs/api/py/webdriver_remote/selenium.webdriver.remote.webdriver.html#module-selenium.webdriver.remote.webdriver